### PR TITLE
[Php80] Copy PhpDocInfo values on ClassPropertyAssignToConstructorPromotionRector

### PR DESCRIPTION
**Pull request by samsonasik**
Closes #743

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/Php80/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector.php [hunk 1]
```diff
@@ -142,6 +146,26 @@ public function provideMinPhpVersion(): int
...
+            return;
+        }
+
+        if ($varTag->description !== '') {
+            return;
+        }
+
+        $phpDocInfo->removeByType(VarTagValueNode::class);
+        $param->setAttribute(AttributeKey::PHP_DOC_INFO, $phpDocInfo);
+    }
```
Comment (TomasVotruba) [inline rules/Php80/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector.php]: Could you extract this to one of existing/new doc-related services?
This should be used in any place related to docs copying.

Comment (samsonasik) [reply rules/Php80/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector.php]: implemented, moved to PhpDocTypeChanger https://github.com/rectorphp/rector-src/pull/744/commits/86a459983eb7c520885ddb17a7a5f3d02e7c2d19

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Thank you :clap:

**Pull request merged**